File: registry-ui/src/app/models/SchemaResource.ts
export class SchemaResource {
  namespace: string;
  contractName: string;
  owner: string;
  version: string;
  schema: any;

  constructor(namespace: string, contractName: string, owner: string, version: string, schema: any) {
    this.namespace = namespace;
    this.contractName = contractName;
    this.owner = owner;
    this.version = version;
    this.schema = schema;
  }
}

File: bindingz-server/src/main/java/io/bindingz/server/controller/SchemaResource.java
/*
 * Copyright (c) 2020 Connor Goulding
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

package io.bindingz.server.controller;

import io.bindingz.server.model.SchemaDto;

public class SchemaResource {

    private final SchemaDto content;

    public SchemaResource(SchemaDto content) {
        this.content = content;
    }

    public SchemaDto getContent() {
        return content;
    }
}
